### DISCUSSION TITLE
Blog TOC Sort

### DISCUSSION THREAD

**Discussion by sheldonhull** [squidfunk/mkdocs-material#5888] [Q&A] [answered]
The sort order by default is oldest post to latest when I have the sidebar show all the posts. Is it possible to reverse this order with a configuration value?

![CleanShot 2023-08-23 at 19 23 49@2x](https://github.com/squidfunk/mkdocs-material/assets/3526320/39fc2f2e-a00b-47ab-b2d4-5eee8fa5b4c0)

**Comment by squidfunk** [comment]
I'm confused. Are you using the [blog plugin](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#built-in-blog-plugin) or a homebrew blog setup? In any case, if you could create a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) that we could inspect, it's much easier to give advice, because it might be related to too many factors: configuration, plugins, overrides, etc.

**Reply by sheldonhull** [reply]
I’m sorry for being unclear. I’m using the new blog feature. Nothing third party. The sidebar that lists the posts (blog_toc I think) is sitting oldest posts first instead of most recent. 

If that isn’t clear enough I’ll publish a branch version so it can be reviewed. Cheers

**Reply by squidfunk** [reply, marked as answer]
That looks like an error. Please create a new issue and provide a minimal reproduction, we'll look into it.

**Comment by ydkmlt84** [comment]
Mine is sorted as newest first, and then to oldest. I want it the other way around. I was thinking maybe I need the pagination function enabled, but I am not sure. Any help would be appreciated! Thanks!

**Reply by squidfunk** [reply]
Please provide a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/), so we can look into it ☺️

**Reply by ydkmlt84** [reply]
Can this be done in docker?

**Reply by squidfunk** [reply]
The minimal reproduction? Sure. The info plugin is available in the Docker image.

**Reply by ydkmlt84** [reply]
I assume I would have to use docker exec? I can do that; I just didn't want to overcomplicate it if I didn't have to.
